### PR TITLE
[RFR]Copostgresqueries

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

**Comment by djhi** [comment]
:+1:

File: e2e/api/setup.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 import config from 'config';
-import dbClient from '../../src/api/lib/db/client';
+import { PgPool } from 'co-postgres-queries';
 import fixturesFactory from '../lib/fixturesLoader';
 import request from '../lib/request';
 
 before(function* setGlobalForTest() {
-    global.db = yield dbClient(config.apps.api.db);
-    global.fixtureLoader = fixturesFactory(global.db.client);
+    global.pool = new PgPool(config.apps.api.db);
```
Comment (brikou) [inline e2e/api/setup.js]: We had a talk with @djhi @fzaninotto @alexisjanvier about the best way to share variables... this is another way :)

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "co": "4.6.0",
     "co-body": "4.2.0",
     "co-pg": "2.0.0",
```
Comment (brikou) [inline package.json]: You definitively should get rid of co-pg and use pg directly

Comment (ThieryMichel) [inline package.json]: It is already the case, just forgot to remove the dependency.

File: e2e/api/setup.js (hunk 1 above)
Comment (djhi) [inline e2e/api/setup.js]: Nooo, please no globals !

**Comment by fzaninotto** [comment]
needs rebase

**Comment by ThieryMichel** [comment]
rebased

File: e2e/api/setup.js (hunk 1 above)
Comment (fzaninotto) [inline e2e/api/setup.js]: To be done in a future PR

**Comment by fzaninotto** [comment]
I love it when you remove lots of code!

**Pull request merged**